src/views/design/ScriptsOverview/index.tsx

```diff
@@ -195,17 +203,46 @@ const ScriptsOverview = withStyles(styles)(
                                     </Box>
                                     {checkAuthorityGeneral(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE)
                                         ? (
-                                            <Box flex="0 0 auto">
-                                                <Button
-                                                    variant="contained"
-                                                    color="secondary"
-                                                    size="small"
-                                                    startIcon={<AddRounded />}
-                                                    onClick={() => redirectTo({ routeKey: ROUTE_KEYS.R_SCRIPT_NEW })}
-                                                >
-                                                    <Translate msg="scripts.overview.header.add_button" />
-                                                </Button>
+                                            <Box display="flex" alignItems="center">
+                                                <Box flex="0 0 auto" mr="8px" width="220px">
+                                                    <Tooltip
+                                                        title="Transform an OpenAPI documentation into IESI concept"
```

Comment:
I just have seen how to render the translation without rendering the Translate component, I will change this line later